endpoints.go

```diff
@@ -61,14 +63,18 @@ var (
 	EndpointUser               = func(uID string) string { return EndpointUsers + uID }
 	EndpointUserAvatar         = func(uID, aID string) string { return EndpointCDNAvatars + uID + "/" + aID + ".png" }
 	EndpointUserAvatarAnimated = func(uID, aID string) string { return EndpointCDNAvatars + uID + "/" + aID + ".gif" }
-	EndpointUserSettings       = func(uID string) string { return EndpointUsers + uID + "/settings" }
-	EndpointUserGuilds         = func(uID string) string { return EndpointUsers + uID + "/guilds" }
-	EndpointUserGuild          = func(uID, gID string) string { return EndpointUsers + uID + "/guilds/" + gID }
-	EndpointUserGuildSettings  = func(uID, gID string) string { return EndpointUsers + uID + "/guilds/" + gID + "/settings" }
-	EndpointUserChannels       = func(uID string) string { return EndpointUsers + uID + "/channels" }
-	EndpointUserDevices        = func(uID string) string { return EndpointUsers + uID + "/devices" }
-	EndpointUserConnections    = func(uID string) string { return EndpointUsers + uID + "/connections" }
-	EndpointUserNotes          = func(uID string) string { return EndpointUsers + "@me/notes/" + uID }
+	EndpointDefaultUserAvatar  = func(uDiscriminator string) string {
+		uDiscriminatorInt, _ := strconv.Atoi(uDiscriminator)
+		return EndpointCDN + "embed/avatars/" + strconv.Itoa(uDiscriminatorInt%5) + ".png"
```

Comment:
Are you missing the uID here somewhere? embed/avatars/1682.png doesn't seem right for my avatar?

Reply:
It's the default avatar url, which is used if you don't have a custom avatar set. And it's discriminator modulo 5, not just the discriminator in the url. 1682 modulo 5 = 2: Your default avatar is https://cdn.discordapp.com/embed/avatars/2.png .

Reply:
There are in total five default avatars:
https://cdn.discordapp.com/embed/avatars/0.png
https://cdn.discordapp.com/embed/avatars/1.png
https://cdn.discordapp.com/embed/avatars/2.png
https://cdn.discordapp.com/embed/avatars/3.png
https://cdn.discordapp.com/embed/avatars/4.png

Reply:
I think the endpoint should be named EndpointUserAvatarDefault so it sits next to the rest of the avatar endpoints.